Firestore/Source/API/FIRFirestoreSettings.mm

```diff
@@ -26,7 +26,7 @@
 static const int64_t kDefaultCacheSizeBytes = 100 * 1024 * 1024;
 static const int64_t kMinimumCacheSizeBytes = 1 * 1024 * 1024;
 // TODO(b/73820332): flip the default.
```

Comment:
Drop the comment? I've updated the linked bug.

Reply:
Oops, thanks. 😁